core/src/test/scala/integration/kafka/api/BaseConsumerTest.scala

```diff
@@ -68,17 +68,17 @@ abstract class BaseConsumerTest extends IntegrationTestHarness with Logging {
     val numRecords = 10000
     sendRecords(numRecords)
 
-    assertEquals(0, this.consumers(0).assignment.size)
-    this.consumers(0).assign(List(tp).asJava)
-    assertEquals(1, this.consumers(0).assignment.size)
+    assertEquals(0, this.consumers.head.assignment.size)
```

Comment:
Changing `apply(0)` to `head` in tests is probably not worth it in the future as the readability improvement is minimal (the latter is even longer). It's OK this time.